drv_priv.h

```diff
@@ -65,7 +65,11 @@ enum CIV_GPU_TYPE {
 	ONE_GPU_VIRTIO,
 	TWO_GPU_IGPU_VIRTIO,
 	TWO_GPU_IGPU_DGPU,
-	THREE_GPU_IGPU_VIRTIO_DGPU
+	THREE_GPU_IGPU_VIRTIO_DGPU,
+	THREE_GPU_IGPU_VIRTIO_IVSHM,
+	FOUR_GPU_IGPU_VIRTIO_DGPU_IVHM,
+	FOUR_GPU_IGPU_VIRTIO_TWO_IVHM,
+	FIVE_GPU_IGPU_VIRTIO_DGPU_TWO_IVHM
 };
```

Comment:
I feel like the code here is getting a little complicated. Maybe use another high bit flag to distinguish the ivsh/non-ivsh status if that is needed, like: GPU_WITH_IVSHMEM = (1<<16) ? Also, "gpu_grp_type" should make a member of cros_gralloc_driver, as in func "reload()" it's not initialized.

Reply:
Yes, you are right. So far the gpu_grp_type is only used to identify whether it has dual gpu, so i modify this logic, please check.
Thanks！

Reply:
> I feel like the code here is getting a little complicated. Maybe use another high bit flag to distinguish the ivsh/non-ivsh status if that is needed, like: GPU_WITH_IVSHMEM = (1<<16) ? Also, "gpu_grp_type" should make a member of cros_gralloc_driver, as in func "reload()" it's not initialized.

Agree, making `gpu_grp_type` a member of `cros_gralloc_driver` would be better. Otherwise we have a few instances of `gpu_grp_type` and should make them consistent.